pages/contacts/_edit.vue

```diff
@@ -343,35 +340,40 @@ export default {
   },
   data() {
     return {
-      contactId   : this.$route.params.edit || '',
-      contact     : {},
-      inputBoxId  : 0,
-      titleOptions: [ ...TITLE_OPTIONS ],
+      contactId    : this.$route.params.edit || '',
+      contact      : {},
+      inputBoxId   : 0,
+      titleOptions : [ ...TITLE_OPTIONS ],
+      emailRequired: false,
     };
   },
   computed: {
-    firstNameRequired() { return this.contact.firstName !== ''; },
-    lastNameRequired() { return this.contact.lastName !== ''; },
-    countyRequired() { return !this.contact.country && !this.contact.isUseOrganizationAddress; },
+    firstNameRequired() { return (this.contact.firstName || '').trim() !== ''; },
+    lastNameRequired() { return (this.contact.lastName || '').trim() !== ''; },
```

Comment:
FirstName and LastName are always required. I think the field are badly named. 
```suggestion
    firstNameValid() { return (this.contact.firstName || '').trim() !== ''; },
    lastNameValid() { return (this.contact.lastName || '').trim() !== ''; },
```

Reply:
DONE ! renamed name as suggested